core/src/main/scala/chisel3/Layer.scala

```diff
@@ -27,16 +27,56 @@ object layer {
     final case object Bind extends Type
   }
 
+  /** Enumeration of different application binary interfaces (ABIs) for how to
+    * enable layers.  These are implementations of the FIRRTL ABI Specification.
+    */
+  private[chisel3] object ABI {
```

Comment:
This is notionally package private, but it's a public def of `trait LayerConfig`. I think it's a bug in Scala that this is allowed but we probably should either make those defs package private or make this public.

Reply:
Interesting discussion about this on Scala discord. 👀 

There isn't really a good reason to make this private. I made it public in: c61fb86a4